### PR TITLE
Upgrade the Gradle wrapper to 8.7, and the Gradle plugin to 0.7.0

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
Could you tell me how to fix this warning?

```
org.embulk.input.TestCommandFileInputPlugin > testShell STANDARD_ERROR
    Hidden dependencies are uninitialized. Maybe using classes loaded by Embulk's top-level ClassLoader.
    SLF4J: No SLF4J providers were found.
    SLF4J: Defaulting to no-operation (NOP) logger implementation
    SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.

```

### CODE REVIEW

**Comment by dmikurube** [comment]
That warning has no problems. It means that the code is writing logs by SLF4J, but no actual logging driver is found, then it is logging nothing out.

If you wanted to elimiate this warning, and write logs in tests, you may want to include Logback in the test dependencies.

**Review by dmikurube — COMMENTED** [review]
When making a pull request (or split multiple pull requests), it'd be nice to keep *each pull request* to pass compile and tests. Unfortunately, this pull request does not compile, and pass the tests.

To keep each pull request to pass compile and tests, you may want to have the `check` GitHub Action at first.

**Comment by dmikurube** [comment]
Made `check` in #17.

**Comment by hiroyuki-sato** [comment]
Thanks. After merging #17. I'll modify something and push again. As a result, I expect CI will run.

**Comment by hiroyuki-sato** [comment]
Due to I haven't changed the Java code yet, CI (#11, #12) fail. I fixed the source codes in #13. It passed CI test.

**Comment by dmikurube** [comment]
CI should pass in each pull request. It is prioritized than splitting pull requests for readability. If this #11 needs code changes in #13, please consider :

* Keep librarires old in this pull request so that this pull request focuses only on the Gradle wrapper and the Gradle plugin.
* Mix all #11, #12, and #13 in the same single pull request.

**Comment by dmikurube** [comment]
Thanks. Allow me to have some time to review this...

File: build.gradle [hunk 1]
```diff
@@ -32,11 +32,14 @@ java {
 }
 
 dependencies {
+
```
Comment (dmikurube) [inline build.gradle]: Unnecessary newline?

File: build.gradle [hunk 2]
```diff
@@ -32,11 +32,14 @@ java {
...
     compileOnly  "org.embulk:embulk-spi:0.11"
     implementation "org.embulk:embulk-util-config:0.5.0"
     implementation "org.embulk:embulk-util-file:0.2.0"
 
     testImplementation "junit:junit:4.13.2"
+    testImplementation "org.embulk:embulk-deps:0.11.3"
```
Comment (dmikurube) [inline build.gradle:41]: We may want to have `embulk-core` as well, explicitly.

File: src/main/java/org/embulk/input/CommandFileInputPlugin.java [hunk 3]
```diff
@@ -123,18 +122,17 @@ public TransactionalFileInput open(TaskSource taskSource, int taskIndex)
...
             }
         } catch (IOException ex) {
-            throw Throwables.propagate(ex);
+            throw new RuntimeException(ex);
         }
     }
 
-    @VisibleForTesting
-    static List<String> buildShell()
+    protected static List<String> buildShell()
```
Comment (dmikurube) [inline src/main/java/org/embulk/input/CommandFileInputPlugin.java]: `protected` needed?

**Review by dmikurube — APPROVED** [review]
Basically LGTM, then approved, but left a couple of nitpicking comments. Thanks for working on it!

File: src/main/java/org/embulk/input/CommandFileInputPlugin.java (hunk 3 above)
Comment (hiroyuki-sato) [reply src/main/java/org/embulk/input/CommandFileInputPlugin.java]: Thanks I wanted to ask how to remove the `@VisibleForTesting` annotation. Fixed.

File: build.gradle (hunk 1 above)
Comment (hiroyuki-sato) [reply build.gradle]: Thanks Fixed.

File: build.gradle (hunk 2 above)
Comment (hiroyuki-sato) [reply build.gradle:41]: `org.embulk:embulk-core` added.

**Pull request merged**